internal/blockutils/cmd.go

```diff
@@ -0,0 +1,31 @@
+package blockutils
+
+import (
+	"bufio"
+	"gobar/internal/log"
+	"os/exec"
+)
+
+// RunCmdStdout runs a command, then captures and returns the output
+func RunCmdStdout(cmd *exec.Cmd) ([]string, error) {
+	var stdoutLines []string
+	stdout, err := cmd.StdoutPipe()
+	if err != nil {
+		return nil, err
+	}
+	defer stdout.Close()
+	if err := cmd.Start(); err != nil {
+		log.FileLog(err)
+		return nil, err
+	}
+
+	sc := bufio.NewScanner(stdout)
+
+	for sc.Scan() {
+		stdoutLines = append(stdoutLines, sc.Text())
+	}
+
+	cmd.Wait()
```

Comment:
the error return from this will tell you that the world is misconfigured or  execution failed some way you could fix